src/utils/asType.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+
+export default function asType<T, SubT: T>(obj: T, klass: Class<SubT>): SubT {
```

Comment:
@jquense Do you like this? I don't love it lol

Reply:
TBH i just copied it without looking at it. Is this mostly a flow hack thing?

Reply:
It's @itajaja's thing. It's a cast in Flow plus a runtime type check.

Reply:
I LOVE IT 🙄

Reply:
also a fan

Reply:
sounds good then